### PR TITLE
feat: add new api for download daily wellness data

### PR DESCRIPTION
**Pull request by Eleven-CH**
add new api for download daily wellness data

### CODE REVIEW

File: src/garmin/GarminConnect.ts [hunk 1]
```diff
@@ -206,6 +206,20 @@ export default class GarminConnect {
         );
     }
 
+    async downloadWellnessData(date: Date, dir: string) {
```
Comment (gregegan) [inline src/garmin/GarminConnect.ts:209]: can you add try / catch handling?

File: README.md [hunk 2]
```diff
@@ -276,6 +276,17 @@ await GCClient.deleteImage(
 );
 ```
 
+### Download wellness data
```
Comment (gregegan) [inline README.md:279]: Please use this readme format for new function definitions

https://github.com/Pythe1337N/garmin-connect/blob/master/README.md#getsleepdatadate-string-promisesleepdata

Comment (gregegan) [reply README.md:279]: reference this post https://github.com/Pythe1337N/garmin-connect/issues/72

File: src/garmin/GarminConnect.ts (hunk 1 above)
Comment (Eleven-CH) [reply src/garmin/GarminConnect.ts]: fixed

**Pull request closed without merging**